### ISSUE TITLE
MS Project Issues task2

### ISSUE THREAD

**Issue by arghyakusumdas** [arghyakusumdas/FourBytes#4]
1) The customer survey cannot be a subtask of the Team meeting
2) Add a subtask called 'wait on hardware arrival' and assign some elapsed time. Since it is elapsed time it should not jeopardize other tasks. The elapsed time must satisfy the deadline
3) Since you are hiring new employees, add a separate task for hiring new employees and add at least one subtask called interview new employees to it.

**Comment by Thegoodjohn** [comment]
Fixed all Issues and made sure there were no scheduling errors and was completed by deadline as well.

**Comment by arghyakusumdas** [comment]
Hire new employee is added as a subtask of 'Ask for quotation'. Try to avoid these mistakes

**Comment by arghyakusumdas** [comment]
Hire new employee is added as a subtask of 'Ask for quotation'. Try to avoid these mistakes

**Comment by Thegoodjohn** [comment]
Fixed.
